data/50-default.json

```diff
@@ -85,5 +85,29 @@
     "ref-kind": "runtime",
     "name": "com.endlessm.CompanionAppService.Locale",
     "branch": "stable"
+  },
+  {
+    "action": "install",
+    "serial": 2019102100,
+    "ref-kind": "app",
+    "collection-id": "com.endlessm.Apps",
+    "remote": "eos-apps",
+    "name": "com.hack_computer.Clubhouse",
+    "branch": "stable",
+    "filters": {
+        "architecture": ["amd64"]
+    }
+  },
+  {
+    "action": "uninstall",
+    "serial": 2019102200,
+    "ref-kind": "app",
+    "collection-id": "com.endlessm.Apps.Hack",
+    "remote": "hack-apps",
+    "name": "com.endlessm.Clubhouse",
+    "branch": "stable",
+    "filters": {
```

Comment:
Again – I think we should just drop the filter here. We don't want `com.endlessm.Clubhouse` to be installed anywhere.